### ISSUE TITLE
Requirements are broken

### ISSUE THREAD

**Issue by maresb** [conda-forge/responses-feedstock#33]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

```bash
docker run --rm -it mambaorg/micromamba
micromamba install -c conda-forge "urllib3<2" "responses"
pip check
```

```
responses 0.23.2 has requirement urllib3<3.0,>=2.0.0, but you have urllib3 1.26.15.
```

### Installed packages

```shell
x
```

### Environment info

```shell
x
```

**Comment by maresb** [comment]
Luckily this is limited to 0.23.2, see first bullet point of https://github.com/getsentry/responses/releases/tag/0.23.2

**Comment by maresb** [comment]
Closed by #35, #34, and https://github.com/conda-forge/admin-requests/pull/771
